helpers/messages/sendMessage.ts

```diff
@@ -69,7 +69,14 @@ export async function sendMessage(bot: Bot, channelId: bigint, content: CreateMe
           replied_user: content.allowedMentions?.repliedUser,
         }
         : undefined,
-      file: content.file,
```

Comment:
Unrelated change please revert this.
Also please combine #2051 into this PR or the other way around.

Reply:
I just move it down, I reorder stuff so that it's in the order discord docs looks like at https://discord.com/developers/docs/resources/channel#embed-limits-jsonform-params :(
![image](https://user-images.githubusercontent.com/87189679/158124818-da9ac85d-a727-4837-8269-d5ed5d049a1c.png)